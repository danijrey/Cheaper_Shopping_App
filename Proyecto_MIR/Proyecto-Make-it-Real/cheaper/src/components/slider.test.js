import React from 'react';
import {render} from '@testing-library/react';
import Slider from './slider';


describe ('Slider',() =>{
    it ('should render correctly', ()=>{
    const {container} = render (<Slider/>); 
    expect (container.firstChild).toMatchSnapshot();
    });
});