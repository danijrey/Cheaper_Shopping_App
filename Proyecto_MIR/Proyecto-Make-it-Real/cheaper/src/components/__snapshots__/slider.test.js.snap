// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slider should render correctly 1`] = `
<div
  class="container-slider carousel slide"
  data-testid="Slider"
>
  <ol
    class="carousel-indicators"
  >
    <li
      class="active"
    />
    <li />
    <li />
    <li />
    <li />
  </ol>
  <div
    class="carousel-inner"
  >
    <div
      class="active carousel-item"
    >
      <img
        alt="First slide"
        class="Food"
        src="food.png"
      />
      <div
        class="carousel-caption"
      >
        <h3
          class="H3-Food"
        >
          Descuentos hasta del 40%
        </h3>
        <p
          class="P-Food"
        >
          Pagando con tu tarjeta Éxito.
        </p>
      </div>
    </div>
    <div
      class="carousel-item"
    >
      <img
        alt="Third slide"
        class="Fruver"
        src="fruver.png"
      />
      <div
        class="carousel-caption"
      >
        <h3
          class="H3-Fruver"
        >
          Frutas siempre frsecas
        </h3>
        <p
          class="P-Fruver"
        >
          Aprovecha, tenemos promociones todos los Jueves
        </p>
      </div>
    </div>
    <div
      class="carousel-item"
    >
      <img
        alt="Third slide"
        class="Pan"
        src="pan.png"
      />
      <div
        class="carousel-caption"
      >
        <h3
          class="H3-Pan"
        >
          Tenemos sorpresas para ti
        </h3>
        <p
          class="P-Pan"
        >
          Solo en Jumbo, obtienes el 50%.Todos los Sábados
        </p>
      </div>
    </div>
    <div
      class="carousel-item"
    >
      <img
        alt="First slide"
        class="Chocolate"
        src="chocolate.png"
      />
      <div
        class="carousel-caption"
      >
        <h3
          class="H3-Chocolate"
        >
          Lo que más te gusta
        </h3>
        <p
          class="P-Chocolate"
        >
          Aprovecha la oferta en chocolates Suizos.
        </p>
      </div>
    </div>
    <div
      class="carousel-item"
    >
      <img
        alt="First slide"
        class="Carne"
        src="CarneSlider.jpg"
      />
      <div
        class="carousel-caption"
      >
        <h3
          class="H3-Carne"
        >
          25% De descuento
        </h3>
        <p
          class="P-Carne"
        >
          Todos los Martes en la Vaquita
        </p>
      </div>
    </div>
  </div>
  <a
    class="carousel-control-prev"
    href="#"
    role="button"
  >
    <span
      aria-hidden="true"
      class="carousel-control-prev-icon"
    />
    <span
      class="sr-only"
    >
      Previous
    </span>
  </a>
  <a
    class="carousel-control-next"
    href="#"
    role="button"
  >
    <span
      aria-hidden="true"
      class="carousel-control-next-icon"
    />
    <span
      class="sr-only"
    >
      Next
    </span>
  </a>
</div>
`;
